pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -161,6 +161,17 @@ message BackupDataSpec {
 message RestoreSpanEntry {
   optional roachpb.Span span = 1 [(gogoproto.nullable) = false];
   repeated roachpb.ImportRequest.File files = 2 [(gogoproto.nullable) = false];
+  optional int64 progressIdx = 3 [(gogoproto.nullable) = false];
```

Comment:
what sets this?

Reply:
The coordinator will set this in `restore_job` - it's currently not set right now, but will be in the planning PR.